adaptive_components/lib/src/adaptive_column.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+import 'package:flutter/material.dart';
+import 'adaptive_container.dart';
+
+class AdaptiveColumn extends StatelessWidget {
+  const AdaptiveColumn({
+    this.gutter,
+    this.margin,
+    this.columns,
+    @required this.children,
+  }) : assert(children != null);
+
+  final double margin;
+  final double gutter;
+  final AdaptiveConstraintsColumn columns;
+  final List<AdaptiveContaine> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (BuildContext context, BoxConstraints constraints) {
+        BreakpointSystemEntry _entry = getBreakpointEntry(context);
+        final double _margin = margin ?? _entry.margin;
+        final double _gutter = gutter ?? _entry.gutter;
+        final int _totalColumns =
+            columns?.getAdaptiveConstraintsColumn(context) == 0
+                ? columns.getAdaptiveConstraintsColumn(context)
+                : _entry.columns;
+
+        return Container(
+          color: Colors.grey,
```

Comment:
Why is this set to grey? It seems like it shouldn't have a color

Reply:
Removed, was for testing.